content/blog/freedom.md

```diff
@@ -5,14 +5,14 @@ author: stpeter
 categories: ["Miscellaneous"]
 ---
 
-Freedom is one of our core values in the Jabber/XMPP community. The focus on freedom goes back to [Jeremie Miller](https://xmpp.org/xsf/people/jer.shtml), who invented the base of our technologies in 1998 as a way for people to connect over the Internet without the silos and restrictions enforced by consumer IM services.
+Freedom is one of our core values in the Jabber/XMPP community. The focus on freedom goes back to Jeremie Miller, who invented the base of our technologies in 1998 as a way for people to connect over the Internet without the silos and restrictions enforced by consumer IM services.
 
 Over the years that sense of freedom has broadened and deepened. Consider:
 
 -   Our community has centered on Internet protocols (not a given open-source codebase) in part to give developers the freedom to use whatever license they want: free source, open source, freeware, shareware, commercial, or any combination thereof.
 -   Our protocols are as free as the air, which means anyone may implement them without asking permission or worrying about patents.
 -   Our protocols are pure XML, which means developers may extend the protocols for their own custom functionality. While we standardize [XMPP extensions](https://xmpp.org/extensions) that are of more general interest, developers have the freedom to make their own extensions and not standardize them through the XSF if they don't want to.
--   As [previously mentioned](http://blog.xmpp.org/?p=9), all of our discussion venues and processes are completely open and transparent, giving any individual the freedom to participate.
+-   As previously mentioned, all of our discussion venues and processes are completely open and transparent, giving any individual the freedom to participate.
```

Comment:
Looks like that https://web.archive.org/web/20070218205953/http://blog.xmpp.org/?p=9 would be this page https://xmpp.org/2007/02/be-open/ now?

Reply:
But this is good, right? The new rendering?